tests/stub/routing/scripts/v4x1/router_yielding_no_writers_adb_sequentially.script

```diff
@@ -0,0 +1,14 @@
+!: BOLT #VERSION#
+!: ALLOW RESTART
+!: AUTO RESET
```

Comment:
Auto Reset is a bit of a gamble here because it allows the driver to send a `RESET` between `RUN` and `PULL`.